presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/EnvironmentModule.java

```diff
@@ -68,6 +69,8 @@ public void configure(Binder binder)
 
         environments.addBinding("singlenode-cassandra").to(SinglenodeCassandra.class);
 
+        environments.addBinding("singlenode-hdp3").to(SinglenodeHdp3.class);
```

Comment:
Add this right after `singlenode`.
It's arbitrary, i know, but my thinking is that we have hadoop-ish envs first (singlenodes in some order, then multinodes). 
Later we have hadoop-free ones.